src/test/java/org/embulk/parser/jsonpath/TestJsonpathParserPlugin.java

```diff
@@ -39,6 +39,7 @@
 import static org.junit.Assert.assertFalse;
 import static org.junit.Assert.assertNull;
 import static org.junit.Assert.assertTrue;
+import static org.junit.Assert.assertThrows;
```

Comment:
Could you fix import ordering? swap `assertThrows` place.

```
[ant:checkstyle] [ERROR] embulk-parser-jsonpath/src/test/java/org/embulk/parser/jsonpath/TestJsonpathParserPlugin.java:42: Wrong order for 'org.junit.Assert.assertThrows' import. [ImportOrder]
```

```suggestion
import static org.junit.Assert.assertThrows;
import static org.junit.Assert.assertTrue;
```